<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TQtPhone</class>
 <widget class="QDialog" name="TQtPhone">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Phone</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label_Number">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>170</width>
     <height>22</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>Number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>1
Íùè</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>2
ABC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>3
DEF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>4
GHI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>5
JKL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>6
MNO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>7
PQRS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>8
TUV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>9
WXYZ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Star">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_0">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0
+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Hash">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>#</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_Call">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>170</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Clear">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="Line" name="line_Vertical">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>16</width>
     <height>351</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Exit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
